cmd/swarmctl/secret/create.go

```diff
@@ -24,8 +24,10 @@ var createCmd = &cobra.Command{
 		var (
 			secretData []byte
 			err        error
+			driver     string
 		)
 
+		driver, _ = flags.GetString("driver")
```

Comment:
For consistency with existing code, please check the error, though I admit it's most likely unnecessary.